### PR TITLE
Add enzyme v2.2.0 with npm auto-update config

### LINKED ISSUE #7651: [Request]enzyme
enzyme
https://github.com/airbnb/enzyme
https://www.npmjs.com/package/enzyme
http://airbnb.io/enzyme/
# 

Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by x09326**
PR for #7651 
@dannyxx001 please help me review it, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.
# Profile of the lib
- Git repository (required): https://github.com/airbnb/enzyme 
- Official website (optional, not the repository): http://airbnb.io/enzyme/
- NPM package url (optional): https://www.npmjs.com/package/enzyme
- GitHub / Bitbucket popularity (required):
  - Count of stars: 4658
  - Count of watchers: 131
  - Count of forks: 219
- NPM download stats (optional):
  - Downloads in the last day: 1609
  - Downloads in the last week: 42035
  - Downloads in the last month: 168879
# Essential checklist
- [ ] I'm the author of this library
  - [ ] I would like to add link to the page of this library on CDNJS on website / readme
- [x] This lib was not found on cdnjs repo
- [x] No already exist / duplicated issue and PR
- [x] The lib has notable popularity
  - [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  - [x] More than 500 downloads stats per month on npm registry
- [x] Project has public repository on famous online hosting platform (or been hosted on npm)
# Auto-update checklist
- [x] Has valid tags for each versions (for git auto-update)
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct.
# Git commit checklist
- [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
- [x] The parent of the commit(s) in the PR is not old than 3 days.
- [x] Pull request is sending from a non-master branch with meaningful name.
- [x] Separate unrelated changes into different commits.
- [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
- [x] Close corresponding issue in commit message
- [x] Mention related issue(s), people in commit message, comment.

close #7651, cc @airbnb

### CODE REVIEW

**Comment by dannyxx001** [comment]
@x09326 where is the repository info? You get this skill from @nolsherry ? XDD

**Comment by x09326** [comment]
@dannyxx001 
Sorry, I was in a hurry.

**Comment by dannyxx001** [comment]
@PeterDaveHello I think this PR is OK. Please review again, thanks.

**Comment by PeterDaveHello** [comment]
hmmmm ... I wonder if the testing utils need or should be hosted on cdn, @cdnjs/team-cdnjs what do you think?

File: ajax/libs/enzyme/2.2.0/Debug.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+
+exports.typeName = typeName;
+exports.spaces = spaces;
+exports.indent = indent;
+exports.debugNode = debugNode;
+exports.debugNodes = debugNodes;
+exports.debugInst = debugInst;
+exports.debugInsts = debugInsts;
+
+var _ShallowTraversal = require('./ShallowTraversal');
```
Comment (PeterDaveHello) [inline ajax/libs/enzyme/2.2.0/Debug.js]: I'm also wondering if this lib works in browsers, @x09326 would you please confirm that?

Comment (x09326) [inline ajax/libs/enzyme/2.2.0/Debug.js]: @lelandrichardson
Could you help us figure out this?
Thanks.

Comment (drewfreyling) [inline ajax/libs/enzyme/2.2.0/Debug.js]: That will be ok and users can use a shim if they need to - http://stackoverflow.com/questions/10282075/object-defineproperty-for-all-browsers

**Comment by PeterDaveHello** [comment]
Hello, any updates here? Thanks.

**Comment by PeterDaveHello** [comment]
Any updates here? Thanks!

**Comment by sufuf3** [comment]
@PeterDaveHello I don't think this lib can use in web front-end.
Can we close this PR?
Thank you.

**Comment by PeterDaveHello** [comment]
I think it needs Browserify, let's close it until it's officially supported by its authors.

**Pull request closed without merging**